docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,152 @@
+If non-Hudi tables are present in the metastore, as well, they are not visible to
+the connector.
+
+To configure the Hudi connector, create a catalog properties file, for
+example ``etc/catalog/hudi.properties``, that references the ``hudi``
+connector. Update the ``hive.metastore.uri`` with the URI of your Hive metastore
+Thrift service:
+
+.. code-block:: properties
+
+    connector.name=hudi
+    hive.metastore.uri=thrift://example.net:9083
+
+Additionally, following configuration properties can be set depending on the use-case.
+
+.. list-table:: Hudi configuration properties
+    :widths: 30, 55, 15
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default
+    * - ``hudi.metadata-enabled``
+      - Fetch the list of file names and sizes from metadata rather than storage.
+      - ``false``
+    * - ``hudi.columns-to-hide``
+      - List of column names that will be hidden from the query output.
+        It can be used to hide Hudi meta fields. By default, no fields are hidden.
+      -
+    * - ``hudi.use-metastore-for-partitions``
+      - Fetch partition info from the metastore. If disabled, then the partition
+        info is fetched using Hudi's partition extractor and relative partition path.
+      - ``true``
+    * - ``hudi.parquet.use-column-names``
+      - Access Parquet columns using names from the file. If disabled, then columns
+        are accessed using index.Only applicable to Parquet file format.
+      - ``true``
+    * - ``hudi.min-partition-batch-size``
+      - Minimum number of partitions returned in a single batch.
+      - ``10``
+    * - ``hudi.max-partition-batch-size``
+      - Maximum number of partitions returned in a single batch.
+      - ``100``
+    * - ``hudi.size-based-split-weights-enabled``
+      - Unlike uniform splitting, size-based splitting ensures that each batch of splits
+        has enough data to process. By default, it is enabled to improve performance.
+      - ``true``
+    * - ``hudi.standard-split-weight-size``
+      - The split size corresponding to the standard weight (1.0)
+        when size based split weights are enabled.
+      - ``128MB``
+    * - ``hudi.minimum-assigned-split-weight``
+      - Minimum weight that a split can be assigned
+        when size based split weights are enabled.
+      - ``0.05``
+    * - ``hudi.max-splits-per-second``
+      - Rate at which splits are enqueued for processing.
+        The queue will throttle if this rate limit is breached.
+      - ``Integer.MAX_VALUE``
+    * - ``hudi.max-outstanding-splits``
+      - Maximum outstanding splits in a batch enqueued for processing.
+      - ``1000``
+
+SQL Support
+-----------
+
+Currently, the connector only provides read access to data in the Hudi table that has been synced to
+Hive metastore. Once the catalog has been configured as mentioned above, users can query the tables
+as usual like Hive tables.
+
+Supported Query Types
```

Comment:
Could you please add a section related to friendliness of Hudi to Hive?

Specifically, document how the hive connector could read (if possible) a Hudi table.